frontend/lib/screens/company_student_list/company_student_list.dart

```diff
@@ -16,21 +17,39 @@ class StudentCardsList extends StatefulWidget {
 }
 
 class _StudentCardsListState extends State<StudentCardsList> {
-  List<Student> students;
+  List<Placement> _placements;
   CardController _cardController;
+  Placement _placement;
+  Map<int, List<Student>> recommendationMap = {};
+  int _employer_id = 1;
```

Comment:
@AliceCasali variables in dart are commonly named with a camel case syntax so this would need to become _employerId in the next stories